atomicapp/applogging.py

```diff
@@ -27,15 +27,16 @@
 class customOutputFormatter(logging.Formatter):
     """
     A class that adds 'longerfilename' support to the logging formatter
-    This 'longerfilename' will be filename + parent dir.
+    This 'longerfilename' will be path/to/file.py after the root atomicapp
+    folder.
     """
 
     def format(self, record):
 
         # Add the 'longerfilename' field to the record dict. This is
         # then used by the Formatter in the logging library when
         # formatting the message string.
-        record.longerfilename = '/'.join(record.pathname.split('/')[-2:])
+        record.longerfilename = record.pathname.split('atomicapp')[2].lstrip('/')
```

Comment:
actually, should we use `[-1]` here instead of `[2]` ? 

This also makes assumptions about the filepath.. would be nicer if we could compare the file path with a known path (i.e. the known path of the atomicapp base dir) and chop off the common parts.